.travis.yml

```diff
@@ -3,38 +3,54 @@ node_js:
     - 12.13.0
 dist: bionic
 cache:
-    directories:
-        - "$HOME/.cache"
+  directories:
+    - "$HOME/.cache"
+env:
+  - PORT=8081 REACT_APP_DHIS2_BASE_URL=http://localhost:8080 REACT_APP_CYPRESS=true CYPRESS_EXTERNAL_API=http://localhost:8080 CYPRESS_ROOT_URL=http://localhost:8081 FEEDBACK_REPO=EyeSeeTea/feedback-ci
 before_install:
-    - echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
-    # Install python dependencies
-    - sudo apt-get update
-    - sudo apt-get install python3 python python3-setuptools docker.io docker-compose
-    #  Install d2-docker
-    - git clone https://github.com/EyeSeeTea/d2-docker.git
-    - cd d2-docker/
-    - sudo python3 setup.py install
-    - d2-docker --help
-    # Hack to not be prompted in the terminal
-    - sudo apt-get remove golang-docker-credential-helpers
-    # Start docker service
-    - sudo systemctl unmask docker.service
-    - sudo systemctl unmask docker.socket
-    - sudo systemctl start docker.service
-    # Login to docker
-    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-    # Start docker service
-    - d2-docker start eyeseetea/dhis2-data:2.30-datasync-sender -d
+  - echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
Not necessarily for this PR, but I think sometime we should extract all those code into 3 or 4 isolated `scripts/*.sh` to keep the travis file shorter and more descriptive.

Reply:
Yeah, it began with a simple couple of commands but now it's a gigantic script.